lib/puppetserver/ca/action/migrate.rb

```diff
@@ -0,0 +1,95 @@
+require 'puppetserver/ca/utils/cli_parsing'
+require 'puppetserver/ca/utils/file_system'
+require 'puppetserver/ca/utils/http_client'
+
+module Puppetserver
+  module Ca
+    module Action
+      class Migrate
+        include Puppetserver::Ca::Utils
+        PUPPETSERVER_CA_DIR = '/etc/puppetlabs/puppetserver/ca'
+
+        SUMMARY = "Migrate the CA directory /etc/puppetlabs/puppetserver/ca"
+        BANNER = <<-BANNER
+Usage:
+  puppetserver ca migrate [--help]
+  puppetserver ca migrate [--config PATH]
+
+Description:
+  Migrate an existing CA directory in the puppet directory to puppetserver. This is for
```

Comment:
Feel like it's already in the summary, but I can add it to the description too if you feel strongly.

Reply:
The former should be left vague, as we just look at the setting and then move it. The second one, I agree should be more explicit, especially since it isn't configurable.